clients/src/main/java/org/apache/kafka/common/metrics/MetricConfig.java

```diff
@@ -97,5 +99,13 @@ public MetricConfig recordLevel(Sensor.RecordingLevel recordingLevel) {
         return this;
     }
 
+    public boolean skipReporting() {
```

Comment:
I asked the very same question: https://github.com/apache/kafka/pull/8768#discussion_r433705077. Basically, we already have the same metrics defined as part of the `SelectorMetrics` so the ones added here collides.

Thinking a bit more about this, I wonder if we could use Sensors without adding them to the registry at all.

Reply:
If we are using a sensor to determine throttle time that is different from the one in Selector, we might want to expose  it as a metric anyway. In case of a bug, we want to know this metric, not one in Selector. Perhaps we could use `connections-accepted` instead of `connections-created` or something like that. In any case, `skipReporting` seems odd, so as @dajac  said, using a Sensor that is not added to the metrics registry may be an option too.